packages/paste-core/components/button/stories/index.stories.tsx

```diff
@@ -6,7 +6,7 @@ import {PlusIcon} from '@twilio-paste/icons/esm/PlusIcon';
 import {Button} from '../src';
 import {ButtonVariants, ButtonSizes, ButtonTabIndexes} from '../src/types';
 
-const ButtonSizeOptions = ['default', 'small', 'icon', 'reset'];
+const ButtonSizeOptions = ['', 'default', 'small', 'icon', 'reset'];
```

Comment:
For non-required props its useful to have stories that default to blank. For example, link buttons default to reset size, but we were setting a size in storybook with knobs. 

Setting a size set the font weight to semibold. Reset size was normal.

We didn't have enough coverage for all the options so we didn't catch it. I've added an example of each below for applitools to check going forward